models/keyword_test.go

```diff
@@ -0,0 +1,123 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/gutakk/go-google-scraper/db"
+	"github.com/gutakk/go-google-scraper/tests"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/go-playground/assert.v1"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func TestValidateFileTypeWithValidFileType(t *testing.T) {
+	result := ValidateFileType("text/csv")
+
+	assert.Equal(t, nil, result)
+}
+
+func TestValidateFileTypeWithInvalidFileType(t *testing.T) {
+	result := ValidateFileType("test")
+
+	assert.Equal(t, "File must be CSV format", result.Error())
+}
+
+func TestValidateCSVLengthWithOneRow(t *testing.T) {
+	result := ValidateCSVLength(1)
+
+	assert.Equal(t, nil, result)
+}
+
+func TestValidateCSVLengthWithOneThoudsandRow(t *testing.T) {
```

Comment:
The min/max values might change so it would better to named some tests in a more generic way e.g. `TestValidateCSVLengthWithMaxRowAllowed`

Reply:
Fixed in b02cbc6b0abe3720c279b654eaecdd704f614207